pkg/storage_nodes_count/storage_nodes_count.go

```diff
@@ -0,0 +1,112 @@
+package storagenodescount
+
+import (
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+const (
+	// SncYamlPath is the name of the yaml that will contain the decision matrix for storage nodes
+	SncYamlPath = "storage_nodes_count.yaml"
+	// YamlFilePath is the path where unit tests will find in this directory
+	YamlFilePath = "generator/" + SncYamlPath
+	// TestFilePath is where the unit test in controller_test will find the yaml file
+	TestFilePath = "../../controller/storagecluster/" + SncYamlPath
```

Comment:
Maybe we can use the `pkg/controller/storagecluster/testspec` directory to store this yaml file.